### PR TITLE
Add warning/error messages for bad initialization params

### LINKED ISSUE #1: Add warning/error messages for bad initialization params in Otel Python
This might be a no-op if the warnings/errors exist already 

Bad initialization params include: 
Bad length of access token 
No service name set (non-fatal error, but should log the default)

### PR DESCRIPTION
**Pull request by ocelotl**
Fixes #1

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -45,4 +45,4 @@ jobs:
         TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
       run: |
-        make publish
+        twine updload dist/*
```
Comment (codeboten) [inline .github/workflows/release.yml:48]: typo here

Comment (ocelotl) [reply .github/workflows/release.yml]: Fixed

File: src/opentelemetry/lightstep/configuration.py [hunk 2]
```diff
@@ -12,78 +12,225 @@
...
+
+    # FIXME use entry points (instead of a dictionary) to locate propagator
+    # classes
+    set_global_httptextformat(
+        CompositeHTTPPropagator(
+            [
+                {
+                    "tracecontext": TraceContextHTTPTextFormat,
+                    "b3": B3Format,
+                    "datadog": DatadogFormat,
```
Comment (codeboten) [inline src/opentelemetry/lightstep/configuration.py:182]: Currently only planning on supporting b3 initially.

File: src/opentelemetry/lightstep/configuration.py [hunk 3]
```diff
@@ -12,78 +12,225 @@
...
-    "LS_SATELLITE_URL", _DEFAULT_SATELLITE_URL
+_DEFAULT_LS_SATELLITE_URL = "ingest.lightstep.com:443"
+_DEFAULT_LS_METRICS_URL = "ingest.lightstep.com:443/metrics"
+
+_LS_ACCESS_TOKEN = _env.str("LS_ACCESS_TOKEN", None)
+_LS_SATELLITE_URL = _env.str("LS_SATELLITE_URL", _DEFAULT_LS_SATELLITE_URL)
+_LS_METRICS_URL = _env.str("LS_METRICS_URL", _DEFAULT_LS_METRICS_URL)
+_LS_SERVICE_NAME = _env.str("LS_SERVICE_NAME", None)
+_LS_SERVICE_VERSION = _env.str("LS_SERVICE_VERSION", "unknown")
+_LS_PROPAGATOR = _env.list("LS_PROPAGATOR", ["tracecontext", "b3"])
```
Comment (codeboten) [inline src/opentelemetry/lightstep/configuration.py:47]: Only supporting b3 by default for now
```suggestion
_LS_PROPAGATOR = _env.list("LS_PROPAGATOR", ["b3"])
```

File: src/opentelemetry/lightstep/configuration.py [hunk 4]
```diff
@@ -12,78 +12,225 @@
...
+        metrics_url (str): LS_METRICS_URL, the URL of the metrics collector
+            where the metrics are to be exported. Defaults to
+            `ingest.lightstep.com:443/metrics`.
+        service_name (str): LS_SERVICE_NAME, the name of the service that is
+            used along with the access token to send spans to the Lighstep
+            satellite. This configuration value is mandatory.
+        service_version (str): LS_SERVICE_VERSION, the version of the service
+            used to sernd spans to the Lightstep satellite. Defaults to
+            `"unknown"`.
+        propagator (str): LS_PROPAGATOR, a list of propagators to be used.
```
Comment (codeboten) [inline src/opentelemetry/lightstep/configuration.py:107]:
```suggestion
        propagator (list): LS_PROPAGATOR, a list of propagators to be used.
```

File: src/opentelemetry/lightstep/configuration.py [hunk 5]
```diff
@@ -12,78 +12,225 @@
...
+            Defaults to `["tracecontext", "b3"]`.
```
Comment (codeboten) [inline src/opentelemetry/lightstep/configuration.py:108]:
```suggestion
            Defaults to `["b3"]`.
```

File: requirements-test.txt [hunk 6]
```diff
@@ -1,2 +1,3 @@
 pytest==5.4.3
 pytest-cov==2.9.0
+ipdb
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply requirements-test.txt:3]: Removed

File: src/opentelemetry/lightstep/configuration.py (hunk 2 above)
Comment (ocelotl) [reply src/opentelemetry/lightstep/configuration.py:182]: Fixed

**Pull request merged**